### ISSUE TITLE
First UAC intro automatically skipped?

### ISSUE THREAD

**Issue by Hexaae** [MadDeCoDeR/Classic-RBDOOM-3-BFG#62]
When starting a New Game, first video about "UAC" is abruptly interrupted (?)
I can see just very few frames of the planet surface and then video is aborted automatically...
https://youtu.be/lnFlBDruFyA?t=10 (starting from 00:10)

Is it by design in DOOM BFA or a bug? Can I "restore" it?
I start the BFG game base files with "+set sys_lang "italian" +set com_game_mode 3" custscenes etc. play fine in Italian language...
EDIT: tested also in english and still the same. Doesn't happen using RBDOOM 3 BFG port so it looks like a (mis)feature of this DOOM BFA edition...

(Using Doom 3 BFG base)

**Comment by MadDeCoDeR** [comment]
There is a feature that allows you to skip the intro text. The skip happens if the game register any input while the intro text is playing. In some (i guess) rare situations it might register an event and skip it without realizing it.

**Comment by Hexaae** [comment]
I'm doing nothing (key/gamepad presses)....

**Comment by Hexaae** [comment]
Wait. I've found the cause: when I press with LMB "Marine" difficulty to start the game it skips the UAC video. Doesn't happen if I press RETURN on keyboard to confirm "Marine"...

Another issue:
...But I've found another issue now: if I use RETURN and load for the first time the UAC intro it will be in English even if I use _+set sys_lang "italian". After the game starts, I quit back to menu, and restart a new game it will be in Italian as expected.
